uni/lib/model/entities/exam.dart

```diff
@@ -134,41 +116,21 @@ class Exam {
       other is Exam &&
           runtimeType == other.runtimeType &&
           subject == other.subject &&
+          listEquals(rooms, other.rooms) &&
           begin == other.begin &&
           end == other.end &&
-          listEquals(rooms, other.rooms) &&
-          day == other.day &&
-          examType == other.examType &&
-          weekDay == other.weekDay &&
-          month == other.month &&
-          year == other.year;
+          type == other.type;
 
   @override
   int get hashCode =>
-      subject.hashCode ^
       begin.hashCode ^
       end.hashCode ^
+      subject.hashCode ^
       rooms.hashCode ^
-      day.hashCode ^
-      examType.hashCode ^
-      weekDay.hashCode ^
-      month.hashCode ^
-      year.hashCode;
-
-  static Map<String, String> getExamTypes() {
-    return _types;
-  }
+      type.hashCode;
```

Comment:
If you want to close #622 :man_shrugging:

Reply:
Sure I can do it.